htdocs/install/mysql/tables/llx_payment_expense_report.sql

```diff
@@ -0,0 +1,26 @@
+-- ===================================================================
+-- Copyright (C) 2015      Alexandre Spangaro   <aspangaro@open-dsi.fr>
+-- Copyright (C) 2023      Gauthier VERDOL      <gauthier.verdol@atm-consulting.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ===================================================================
+
+create table llx_expensereport_payment_expensereport
```

Comment:
Can you remind me the goal of feature:

Being able to pay a common expense report and split payment into n users sharing the same expense ?
If this is the need, why not just adding the field  "fk_paiementuser"  into table llx_payment_expensereport  ?
An expense report can alreaydy be splitted into several payments. So we can store the detail of each payment 

Or being able to make 1 payment for n expense reports of he same user ?

Reply:
Hi @eldy 

The goal is being able to make 1 payment for n expense reports of the same user.